ci/pingcap_tidb_operator_build_kind.groovy

```diff
@@ -161,6 +173,8 @@ def call(BUILD_BRANCH, CREDENTIALS_ID, CODECOV_CREDENTIALS_ID) {
 			container("golang") {
 				def WORKSPACE = pwd()
 				dir("${PROJECT_DIR}") {
+					deleteDir()
```

Comment:
because we reuse `build_go1130_memvolume` nodes to build, previous files of other jobs must be deleted first

failure logs on this PR: https://internal.pingcap.net/idc-jenkins/blue/organizations/jenkins/operator_ghpr_e2e_test_kind/detail/operator_ghpr_e2e_test_kind/2667/pipeline/39
```
2020-02-07T08:38:48.004Z] go vet check

[2020-02-07T08:38:49.901Z] # github.com/pingcap/tidb-operator/pkg/client/listers/pingcap/v1alpha1

[2020-02-07T08:38:49.901Z] pkg/client/listers/pingcap/v1alpha1/tidbclusterautoscaler.go:28:41: undefined: v1alpha1.TidbClusterAutoScaler

[2020-02-07T08:38:49.901Z] pkg/client/listers/pingcap/v1alpha1/tidbclusterautoscaler.go:31:2: undefined: TidbClusterAutoScalerListerExpansion

[2020-02-07T08:38:49.901Z] pkg/client/listers/pingcap/v1alpha1/tidbclusterautoscaler.go:45:78: undefined: v1alpha1.TidbClusterAutoScaler

[2020-02-07T08:38:49.901Z] pkg/client/listers/pingcap/v1alpha1/tidbclusterautoscaler.go:60:41: undefined: v1alpha1.TidbClusterAutoScaler

[2020-02-07T08:38:49.901Z] pkg/client/listers/pingcap/v1alpha1/tidbclusterautoscaler.go:62:21: undefined: v1alpha1.TidbClusterAutoScaler

[2020-02-07T08:38:49.901Z] pkg/client/listers/pingcap/v1alpha1/tidbclusterautoscaler.go:63:2: undefined: TidbClusterAutoScalerNamespaceListerExpansion

[2020-02-07T08:38:49.901Z] pkg/client/listers/pingcap/v1alpha1/tidbclusterautoscaler.go:74:86: undefined: v1alpha1.TidbClusterAutoScaler

[2020-02-07T08:38:49.901Z] pkg/client/listers/pingcap/v1alpha1/tidbclusterautoscaler.go:82:66: undefined: v1alpha1.TidbClusterAutoScaler
```